plugin/adsk/scripts/mayaUsdTranslatorExport.mel

```diff
@@ -336,10 +446,20 @@ global proc int mayaUsdTranslatorExport (string $parent,
 //  Parameters:
 //    $parent - the elf parent layout for this options layout. It is
 //            always a scrollLayout.
-//    $action - the action that is to be performed with this invocation of this proc.
+//    $actionAndSections - the action that is to be performed with this invocation of this proc.
 //        Valid options are:
-//        "query" - construct the options string and pass it to the resultCallback.
-//        "post" - post all the elf controls.
+//            "query" - construct the options string and pass it to the resultCallback.
+//            "post" - post all the elf controls.
+//
+//        In addition to the action, the list of UI sections that are desired can be added
+//        with the syntax "action=section1;section2;..." where the section names can be:
+//            context, output, geometry, materials, animation, advanced
+//
+//        The section name can also be "all", "none" or prefixed with "!":
+//            - When "all" is used, all sections are included.
+//            - When "none" is used, all sections are removed.
+//            - When a "!" prefix is used, the sectionis removed instead of added.
+//

```

Comment:
This is how the section are selected. By default all section are present. Sections can be all removed or used with "none" and "all". Otherwise individual ones are added by name or removed by prefixing their name with ! For example:

- "none;geometry;animations" would keep only the Geometry and Animation sections.
- "all;!geometry" would remove geometry. So would the simpler "!geometry" since by default we keep all sections.